src/commands/gitleaks.yaml

```diff
@@ -1,113 +1,28 @@
 parameters:
-  branch:
-    description: "The branch to audit."
-    type: "string"
-  commit:
-    description: "Scan all files changed by this commit (rev. SHA)."
-    type: "string"
-  commit-from:
-    description: "Commit to start audit from (rev. SHA). This is the most recent commit to audit. Default is the latest."
-    type: "string"
   commit-to:
     description: "Commit to stop audit (rev. SHA). This is the oldest commit to audit. Default is the root commit. \
     In CircleCI, you can use << pipeline.git.base_revision >> to scan only since the parent commit."
     type: "string"
```

Comment:
Why not using `<< pipeline.git.base_revision >>` and removing this param?

Reply:
You can't specify a pipeline variable inside the actual orb, only from inside a workflow. So this has to be there in order to pass the base revision from the workflow to the orb. 

If you're curious, CircleCI says they're working on a replacement, but they don't support pipeline variables in orbs because it would make builds un-reproduceable since subsequent runs of a job with the same configuration might result in different orb behavior.

Reply:
I see. That sucks balls. Let's please add an example to the README.md.

Reply:
I will remove this as well. In the absence of using different configs for branches or different logic this isn't useful.